dev/requirements.txt

```diff
@@ -10,8 +10,10 @@ scipy>=1.7.3
 statsmodels>=0.13.2
 tabulate
 plotly
+ray<2.5.0
```

Comment:
I think we don't have this restriction anymore

Reply:
@jmoralez fixed